src/axom/mir/clipping/ClipCases.h

```diff
@@ -88,87 +90,87 @@ namespace visit {
 #define NOCOLOR 122
 
 // Tables
-extern VISIT_VTK_LIGHT_API int numClipCasesHex;
```

Comment:
Thanks for fixing the export macro. I don't know how I overlooked that.

Reply:
(there were a lot of files :) )